.circleci/config.yml

```diff
@@ -639,28 +639,22 @@ workflows:
   version: 2
   go-tests:
     jobs:
-      - check-vendor:
-          filters:
-            branches:
-              ignore:
-                - stable-website
-                - /^docs\/.*/
-                - /^ui\/.*/
-      - lint-consul-retry:
+      - check-vendor: &filter-non-go
```

Comment:
Nit: It would be nice to give this a slightly longer name so that it reads a bit better. Right now it reads as if we are only running on non-go branches.

Maybe `filter-ignore-non-go-branches` ?

Reply:
good point, I'll make the change!